deploy/kubernetes/heartbeat-kubernetes.yaml

```diff
@@ -100,7 +100,7 @@ spec:
           runAsUser: 0
         resources:
           limits:
-            memory: 1536mi
+            memory: 1536Mi
```

Comment:
it should be a string as at https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/

Reply:
It is not a requirement from K8S to follow a specific scalar representation for strings: https://helm.sh/docs/chart_template_guide/yaml_techniques/#strings-in-yaml

I wouldn't mind changing it if required, but I'm assuming all k8s templates' keys would have to be updated for consistency.